Nightly build fails with Maven errors

See: https://github.com/lf-lang/lingua-franca/runs/3706769597?check_suite_focus=true

Summary:

```
Error:  Cannot resolve target definition:
Error:    Software being installed: org.eclipse.lsp4e 0.13.5.202103011336
Error:    Software being installed: org.eclipse.wildwebdeveloper.feature.feature.group 0.10.9.202109141642
Error:    Only one of the following can be installed at once: [org.eclipse.lsp4e 0.13.6.202108060851, org.eclipse.lsp4e 0.13.5.202103011336]
Error:    Cannot satisfy dependency: org.eclipse.wildwebdeveloper 0.5.17.202109141642 depends on: osgi.bundle; org.eclipse.lsp4e 0.13.6
Error:    Cannot satisfy dependency: org.eclipse.wildwebdeveloper.feature.feature.group 0.10.9.202109141642 depends on: org.eclipse.equinox.p2.iu; org.eclipse.wildwebdeveloper [0.5.17.202109141642,0.5.17.202109141642]
Error:  
Error:  Failed to resolve target definition /home/runner/work/lingua-franca/lingua-franca/org.lflang.targetplatform/org.lflang.targetplatform.target: See log for details -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
Error: Process completed with exit code 1.
```

--- a/oomph/LinguaFranca.setup
+++ b/oomph/LinguaFranca.setup
@@ -143,7 +143,7 @@
   <setupTask
       xsi:type="setup:VariableTask"
       name="eclipse.target.platform"
-      value="2021-06"
+      value="2021-09"
       storageURI="scope://Workspace"/>
   <setupTask
       xsi:type="setup.p2:P2Task">
@@ -360,9 +360,9 @@
         <repositoryList
             name="DefaultRepositories">
           <repository
-              url="http://download.eclipse.org/releases/2021-06"/>
+              url="http://download.eclipse.org/releases/2021-09"/>
           <repository
-              url="http://download.eclipse.org/tools/orbit/downloads/drops/R20210602031627/repository/"/>
+              url="http://download.eclipse.org/tools/orbit/downloads/drops/R20210825222808/repository/"/>
           <repository
               url="https://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.25.0/"/>
           <repository
@@ -373,8 +373,6 @@
               url="https://www.pydev.org/updates/"/>
           <repository
               url="http://download.eclipse.org/wildwebdeveloper/releases/latest/"/>
-          <repository
-              url="http://download.eclipse.org/tools/ajdt/410/dev/update"/>
         </repositoryList>
       </targlet>
     </setupTask>

--- a/oomph/LinguaFranca.setup
+++ b/oomph/LinguaFranca.setup
@@ -370,9 +370,9 @@
           <repository
               url="https://kieler.github.io/KLighD/v2.1.0/"/>
           <repository
-              url="https://www.pydev.org/updates/"/>
+              url="https://github.com/fabioz/Pydev/releases/download/pydev_9_0_1/"/>
           <repository
-              url="http://download.eclipse.org/wildwebdeveloper/releases/latest/"/>
+              url="https://download.eclipse.org/wildwebdeveloper/releases/0.13.0/"/>
         </repositoryList>
       </targlet>
     </setupTask>

--- a/org.lflang.rca/lflang.product
+++ b/org.lflang.rca/lflang.product
@@ -101,6 +101,7 @@ OF SUCH DAMAGE.
       <feature id="org.eclipse.jdt"/>
       <feature id="org.eclipse.cdt"/>
       <feature id="org.python.pydev.feature"/>
+      <feature id="org.eclipse.wildwebdeveloper.feature"/>
       <feature id="de.cau.cs.kieler.klighd.view.feature"/>
       <feature id="de.cau.cs.kieler.klighd.freehep.feature"/>
       <feature id="org.eclipse.tm.terminal.feature"/>

--- a/org.lflang.targetplatform/org.lflang.targetplatform.target
+++ b/org.lflang.targetplatform/org.lflang.targetplatform.target
@@ -18,7 +18,7 @@
       <unit id="org.aopalliance" version="0.0.0"/>
       <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.tm.terminal.feature.feature.group" version="0.0.0"/>
-      <repository location="http://download.eclipse.org/releases/2021-06"/>
+      <repository location="http://download.eclipse.org/releases/2021-09"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
       <unit id="org.eclipse.xtext.runtime.feature.group" version="0.0.0"/>
@@ -29,7 +29,7 @@
       <unit id="com.google.inject" version="0.0.0"/>
       <unit id="jakarta.activation" version="0.0.0"/>
       <unit id="javax.activation" version="0.0.0"/>
-      <repository location="http://download.eclipse.org/tools/orbit/downloads/drops/R20210602031627/repository/"/>
+      <repository location="http://download.eclipse.org/tools/orbit/downloads/drops/R20210825222808/repository/"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
       <unit id="org.jetbrains.kotlin.feature.feature.group" version="0.0.0"/>

--- a/org.lflang.targetplatform/org.lflang.targetplatform.target
+++ b/org.lflang.targetplatform/org.lflang.targetplatform.target
@@ -33,19 +33,15 @@
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
       <unit id="org.jetbrains.kotlin.feature.feature.group" version="0.0.0"/>
-      <repository location="https://download.jetbrains.com/kotlin/eclipse-plugin/last/"/>
-    </location>
-    <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
-      <unit id="org.eclipse.contribution.weaving.feature.group" version="0.0.0"/>
-      <repository location="http://download.eclipse.org/tools/ajdt/410/dev/update"/>
+      <repository location="https://download.jetbrains.com/kotlin/eclipse-plugin/0.8.21/"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
       <unit id="org.python.pydev.feature.feature.group" version="0.0.0"/>
-      <repository location="https://www.pydev.org/updates/"/>
+      <repository location="https://github.com/fabioz/Pydev/releases/download/pydev_9_0_1/"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
       <unit id="org.eclipse.wildwebdeveloper.feature.feature.group" version="0.0.0"/>
-      <repository location="http://download.eclipse.org/wildwebdeveloper/releases/latest/"/>
+      <repository location="https://download.eclipse.org/wildwebdeveloper/releases/0.13.0/"/>
     </location>
     <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
       <unit id="de.cau.cs.kieler.klighd.feature.feature.group" version="0.0.0"/>

